### ISSUE TITLE
Add Id to tag class

### ISSUE THREAD

**Issue by tfinez** [sdl/dxa-web-application-java#66]
For relating front info with domain information we need the id of the keyword, because we need to get the metadata, and this is not still available, then we query the broker. Querying the broker for an id keyword is very costing and having into account that it's already mapped in the json, it's so simple as to add the Id in the Tag class. I wanted to do a pull request, but i don't have permissions in the repo. 

Our modifications are:

**Tag.java**

Added new field:

```
@JsonProperty("Id")
private String id;
```

**KeywordFieldConverter.java**

Add in function getTagValues

```
tag.setId(keyword.getId());
```

Can you consider to add them in following versions?

**Comment by rpannekoek** [comment]
Your suggestion makes sense.  We already have a story lined up for DXA 1.7 to improve Keyword Model mapping. I will include this idea to the story description.
BTW: AFAIK, everybody is allowed to do Pull Requests on the develop branch.

**Comment by rpannekoek** [comment]
TSI-811 (internal issue ID for tracking purposes)

**Comment by tfinez** [comment]
thank you, i was trying to create a branch for doing the PR from my branch. I will do it from develop for future pulls.

**Comment by rpannekoek** [comment]
Fixed in DXA 1.7: introduced class `KeywordModel` which provides access to the CM Keyword ID and all other Keyword properties.
